Packet++/src/SipLayer.cpp

```diff
@@ -129,7 +130,7 @@ SipRequestFirstLine::SipRequestFirstLine(SipRequestLayer* sipRequest) : m_SipReq
 }
 
 SipRequestFirstLine::SipRequestFirstLine(SipRequestLayer* sipRequest, SipRequestLayer::SipMethod method, std::string version, std::string uri)
-		throw(SipRequestFirstLineException)
+try		// throw(SipRequestFirstLineException)

```

Comment:
Isn't it better to wrap the try-catch clause with curly brackets (in the beginning and end of the constructor?)